models/iosxr/bgp_global/iosxr_bgp_global.yaml

```diff
@@ -0,0 +1,826 @@
+              suboptions:
+                Network_Minimum_prefix_length:
+                  description: A.B.C.D/length or X:X::X/length  Network/Minimum prefix length.
+                  type: dict
+                  suboptions:
+                    value:
+                      description: A.B.C.D/length or X:X::X/length  Network/Minimum prefix length.
+                      type: str
+                    max:
+                      description: Maximum prefix length. Example- <1-128>  .
+                      type: int
+                    origin:
+                      description: Origin Autonomous System number (in asplain format) Example-<1-4294967295>.
+                      type: int
+            server:
+              description: Configure RPKI cache-servers.
+        vrfs:
+          description: Specify a vrf name.
+          type: list
+          elements: dict
+          suboptions:
+            vrf:
+              description: VRF name.
+              type: str
+            bfd: *bfd_params
+            bgp_params:
+              description: BGP commands.
+              type: dict
+              suboptions:
+                auto_policy_soft_reset: *auto_policy_soft_reset
+                bestpath: *bestpath
+                default: *default
+                enforce_first_as: *enforce_first_as
+                fast_external_fallover: *fast_external_fallover
+                log: *bgp_log
+                multipath: *multipath
+                redistribute_internal: *redistribute_internal
+                router_id: *router_id
+                unsafe_ebgp_policy: *ebgp_policy
+            default_information: *default_info
+            default_metric: *default_metric
+            mpls: *mpls
+            neighbor: *neighbors
+            rd:
+              description: route distinguisher.
+              type: dict
+              suboptions:
+                value:
+                  description: 4-byte AS number in asdot (X.Y) format or 2-byte/4-byte AS number or Ip address.
+                  type: str
+                auto:
+                  description: Automatic route distinguisher.
+                  type: bool
+            socket: *socket
+            timers: *timers
+    running_config:
+      description:
+      - This option is used only with state I(parsed).
+      - The value of this option should be the output received from the IOSXR device by
+        executing the command B(show running-config router bgp).
+      - The state I(parsed) reads the configuration from C(running_config) option and
+        transforms it into Ansible structured data as per the resource module's argspec
+        and the value is then returned in the I(parsed) key within the result.
+      type: str
+    state:
+      description:
+      - The state the configuration should be left in.
+      type: str
+      choices: [deleted, merged, overridden, replaced, gathered, rendered, parsed]
+      default: merged
+EXAMPLES:
```

Comment:
The updated builder no longer handles `EXAMPLES`.

Reply:
Ack, made changes